libsolidity/interface/CompilerStack.cpp

```diff
@@ -1467,7 +1467,6 @@ void CompilerStack::generateEVMFromIR(ContractDefinition const& _contract)
 		m_debugInfoSelection
 	);
 	stack.parseAndAnalyze("", compiledContract.yulIROptimized);
-	stack.optimize();
 
 	//cout << yul::AsmPrinter{}(*stack.parserResult()->code) << endl;
```

Comment:
Unrelated, but we have a few of these scattered around; should probably remove them.

Reply:
How to remove